### ISSUE TITLE
CI/CD pipeline

### ISSUE THREAD

**Issue by leobenkel** [leobenkel/zparkio.g8#3]
same as : https://github.com/leobenkel/ZparkIO/issues/88

**Comment by z4f1r0v** [comment]
How about [Github actions](https://docs.github.com/en/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions)?

**Comment by leobenkel** [comment]
Yes that would be great !

**Comment by z4f1r0v** [comment]
@leobenkel, even though https://github.com/leobenkel/ZparkIO/pull/95 was merged nothing will go out to the world until we get a new release pipeline, right?

**Comment by leobenkel** [comment]
yes , we need to push the binary to sonatype for it to be available to download. Can take a look at https://search.maven.org/search?q=g:com.leobenkel%20a:zparkio* to check where we are at

**Comment by leobenkel** [comment]
@z4f1r0v , I am not sure what we were talking about but the ZparkIO version had been updated and moved to github action. But this repo is not on github action yet.
